code/modules/mob/living/carbon/breathe.dm

```diff
@@ -5,6 +5,26 @@
 	if(SSair.times_fired%4==2 || failed_last_breath || (health < config.health_threshold_crit)) 	//First, resolve location and get a breath
 		breathe()
 
+/mob/living/carbon/proc/inhale(var/datum/reagents/from, var/datum/reagents/target, var/amount = 1, var/multiplier = 1, var/copy = 0)
+
+	if(!(species.flags & IS_PLANT)) //Plants don't breath like humans
+		if(species && (species.flags & NO_BREATHE)) //Check for species
+			return 0
+
+		if (contents.Find(internal) && wear_mask && (wear_mask.item_flags & AIRTIGHT)) //Check for internals
+			return 0
+	/* if someone could get this working, I would be grateful.
+	if(last_smell_time + (5 SECONDS) < world.time)
+		var/datum/reagents/temp = new(amount) //temporary holder used to analyse what gets transfered.
```

Comment:
This looks weird. Surely you could just get the taste message from the human's inhaled reagents list?

Reply:
Also, if you don't plan on salvaging this. Remove it and associated variables. Do not commit dead code.

Reply:
I'll comment it out. Perhaps when I'm a robust enough coder, I can implement it.